### PR TITLE
setup.py for pypi

### PR DESCRIPTION
**Pull request by marqh**
proposed replacement for #2544

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -194,11 +194,21 @@ def extract_version():
...
-    name='Iris',
+    name='iris-scitools',
     version=extract_version(),
     url='http://scitools.org.uk/iris/',
-    author='UK Met Office',
-
+    author='Iris, SciTools, UK Met Office',
+    maintainer='Iris, SciTools',
+    license='LGPLv3',
+    platforms='any',
```
Comment (bjlittle) [inline setup.py]: @marqh `platforms='any'` ... really? Commodore64, please!

Comment (QuLogic) [inline setup.py]: Is this even used any more? I though it was more in classifiers.

**Comment by marqh** [comment]
@bjlittle any further requests for change on this?
is it good to merge?

**Comment by pelson** [comment]
I left a review comment at https://github.com/SciTools/iris/pull/2544#discussion_r121434569. Essentially, given http://legacy.python.org/dev/peps/pep-0423/#top-level-namespace-relates-to-code-ownership, I'd favour naming it scitools-iris.

**Comment by marqh** [comment]
> given http://legacy.python.org/dev/peps/pep-0423/#top-level-namespace-relates-to-code-ownership, I'd favour naming it scitools-iris.

0423 seems to be related to namespace in code.  there is no implication here on namespace in code, the namespace for iris is `iris`

this name only affects pypi naming and `pip install` as far as i can see.

`pip install iris-scitools`

seems more intuitive to me than

`pip install scitools-iris`

and i would prefer 

`pip search iris | sort` to group iris-scitools with other irises to aid informed choice on install

how do objectors address the comment from @lbdreyer:

> I think I prefer iris-scitools with 'iris' at the start. If the packages were sorted alphabetically, and someone looked for iris they'd find iris-scitools nearby. 
https://github.com/SciTools/iris/pull/2544#discussion_r118301804

if I were to flipflop, would this just get merged, or would it fire off further "i'd prefer it the other way" comments?

**Comment by pelson** [comment]
Closed by #2890.

**Pull request closed without merging**